exp/kubeadm-ignition/types/README.md

```diff
@@ -0,0 +1,9 @@
+# Kubeadm types
```

Comment:
If these are the same types being used by the cloud-init based bootstrap provider, you might be able to just import those rather than keeping a separate copy here as well.

Reply:
> If these are the same types being used by the cloud-init based bootstrap provider, you might be able to just import those rather than keeping a separate copy here as well.

Right.
I removed unnecessary types, and tried to use existing types outside of `exp`.
Though types related to systemd units should stay here under `exp/kubeadm-ignition/types`.